src/concepts/pipeline/event_pipeline.md

```diff
@@ -9,17 +9,18 @@ fx_code(fa:fa-cog Firefox code) --> firefox(fa:fa-firefox Firefox Telemetry)
 fx_extensions(fa:fa-cog Mozilla extensions) --> firefox
 fx_hybrid(fa:fa-cog Hybrid Content) --> firefox
 mobile(fa:fa-cog Mobile products) --> mobile_telemetry(fa:fa-firefox Mobile Telemetry)
-firefox -->|main ping| pipeline((fa:fa-database Firefox Data Pipeline))
-firefox -.->|events ping, planned| pipeline
+firefox -.->|main ping, Firefox <62| pipeline((fa:fa-database Firefox Data Pipeline))
+firefox -->|event ping, Firefox 62+| pipeline
 mobile_telemetry --> |mobile events ping| pipeline
-pipeline -->|Firefox events| main_summary[fa:fa-bars main summary table]
-main_summary --> events_table[fa:fa-bars events table]
+pipeline -->|Firefox <62 events| main_summary[fa:fa-bars main summary table]
+pipeline -->|Firefox 62+ events| events_table[fa:fa-bars events table]
+main_summary --> events_table
 pipeline -->|Mobile events| mobile_events_table[fa:fa-bars mobile events table]
 main_summary --> redash(fa:fa-bar-chart Redash)
 events_table --> redash
-events_table -.->|planned| amplitude(fa:fa-bar-chart Amplitude)
+events_table -->|on request| amplitude(fa:fa-bar-chart Amplitude)
 mobile_events_table --> redash
-mobile_events_table --> amplitude
+mobile_events_table -->|on request| amplitude
```

Comment:
Technically, both the desktop and mobile events --> amplitude operate on heka-framed pings so there should just be one line, `pipeline -->|on request| amplitude